### DISCUSSION TITLE
Welcome to Trivy👋🏻

### DISCUSSION THREAD

**Discussion by AnaisUrlichs** [aquasecurity/trivy#1888] [Announcements]
We’re using Discussions here on GitHub as a place to connect with other members of our community.

📚Here are a few resources that might be helpful for you to get started with Trivy:
- [The Trivy Documentation](https://aquasecurity.github.io/trivy/)
- [Have a look at the tutorials on our YouTube Channel](https://www.youtube.com/@AquaSecOSS)

👋🏻 Make sure to [join the community!!](https://slack.aquasec.com) and say hi 😊

If you have any questions, feel free to 
- Post in the Discussions 
- Or ask on Slack 

💡We would also love to hear from you about ideas for integrations and improvements, so feel free to reach out and share your thoughts!

**Comment by afdesk** [comment]
The link to [Katacoda examples](https://www.katacoda.com/aquaopensource) doesn't work now.

```
katacoda.com is now closed to the public and content is no longer available.
```
